.travis.yml

```diff
@@ -11,9 +11,17 @@ matrix:
     - os: linux
       dist: trusty
       sudo: required
+    - os: linux
+      dist: trusty
+      sudo: required
+      env: SWIFT_SNAPSHOT=swift-4.2-DEVELOPMENT-SNAPSHOT-2018-07-01-a
     - os: osx
       osx_image: xcode9.2
```

Comment:
Can you change is to Xcode9.4? That version comes with Swift 4.1.2 and stops the need for downloading the binary, speeding up the build.